docs/core/versions/selection.md

```diff
@@ -75,13 +75,13 @@ A given SDK supports a fixed set of frameworks, capped to the target framework o
 
 When you run an application from source with [`dotnet run`](../tools/dotnet-run.md), from a [**framework-dependent deployment**](../deploying/index.md#publish-runtime-dependent) with [`dotnet myapp.dll`](../tools/dotnet.md#description), or from a [**framework-dependent executable**](../deploying/index.md#publish-runtime-dependent) with `myapp.exe`, the `dotnet` executable is the **host** for the application.
 
-The host chooses the latest patch version installed on the machine. For example, if you specified `netcoreapp3.0` in your project file, and `3.0.4` is the latest .NET runtime installed, the `3.0.4` runtime is used.
+The host chooses the latest patch version installed on the machine. For example, if you specified `netcoreapp3.0` in your project file, and `3.0.3` is the latest .NET runtime installed, the `3.0.3` runtime is used.
```

Comment:
Let's change this from 3.0.3 to 3.0.2. I think it's important that the host will use the latest installed patch, even if a newer patch version is available. 
```suggestion
The host chooses the latest patch version installed on the machine. For example, if you specified `netcoreapp3.0` in your project file, and `3.0.2` is the latest .NET runtime installed, the `3.0.2` runtime is used.
```